### PR TITLE
Create HodlTokenClub WL.md

### PR DESCRIPTION
**Pull request by Shamsham01**

### CODE REVIEW

**Comment by Shamsham01** [comment]
@sasurobert any update on this PR please?

**Comment by PABourdais** [comment]
You really think he will review it just by waiting and asking once ?

You need to harass their team for few weeks, good luck

**Comment by Shamsham01** [comment]
Well, I guess the entire team is busy with launch of xPortal app. I hope that they had successful launch now, someone will come back to me. This is 3rd time I applied for WL, to be honest I do not rush anywhere, but it's a bit frustrating answering community questions, wen staking? I have many whales invested large amounts of money into HodlTokenClub and they have a right to benefit from their investment. I have taken my commission (25% of EGLD from minting), entire treasury belong to the investors then (remaining 75% were put into EGLD/MEX LP on xExchange) and they have right to benefit from this position. I hope @LucianMincu  or @sasurobert finally respond to my PR. All I can do is to keep tagging them, there is no need to be aggressive or harass people. Patience is a virtue ;)

**Comment by PABourdais** [comment]
You opened the PR on the 7th February, in less than a week it will make 1 month.
Let see if they can get a new record on this, I bet on more than 2 months, 3 months would be legendary 😍

**Comment by sasurobert** [comment]
Hi @Shamsham01 ! Sorry for responding late. Verifying the PR now.

**Comment by Shamsham01** [comment]
Thank you. No worries! busy times 🙂

File: LKMEX-whitelist/HodlTokenClub WL.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+The whitelisted projects will need to wrap the XMEX they want to deposit or use. By wrapping it, the XMEX will 
+temporarily “lose” its energy but the energy can be recovered and regained by unWrap it back.
+
+Projects will be able to use the wrappedXMEX to deposit and use in the whitelisted distribution Smart Contracts. Users who will
+get/claim wrappedXMEX will need to unWrap it in order to gain its original energy and convert it back to XMEX 
+in order to enter liquidity or farms.
+
+wrappedXMEX does not have the benefits of the XMEX. Only after unwrapping you gain all the benefits.
+
```
Comment (sasurobert) [inline LKMEX-whitelist/HodlTokenClub WL.md]: delete the first 38 lines.

File: LKMEX-whitelist/HodlTokenClub WL.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    xmex_wrapper_sc-->>rewards_depositer_account: remove energy & transfer wXMEX
+    rewards_depositer_account->>distribution_sc: deposit wXMEX
+    user_account->>distribution_sc: claim wXMEX
+    rect rgba(191, 223, 255, .1)
+    note right of xmex_wrapper_sc: Integrated in xExchange
+    user_account->>xmex_wrapper_sc: unwrapLockedToken wXMEX
+    xmex_wrapper_sc-->>user_account: add energy & transfer XMEX
+    end
+```
+
```
Comment (sasurobert) [inline LKMEX-whitelist/HodlTokenClub WL.md]: you can delete all the lines until here.

Comment (Shamsham01) [reply LKMEX-whitelist/HodlTokenClub WL.md]: Done. all requested lines deleted @sasurobert

**Comment by Shamsham01** [comment]
All Done! @sasurobert

**Comment by jacobleygonie** [comment]
Hi @Shamsham01 !

A remark related to the contract's security. If I am not mistaking, the `stake` endpoint does not verify the token identifier. Consequently:

- Attacker Alice can spam the contract with fake NFTs and take over all rewards.

- Attacker Alice can stake fake NFTs with specific nonces, which match the nonces of genuine NFTs staked by other users. By calling `unstake` she can then drain all the NFTs of other users.

**Comment by Shamsham01** [comment]
Thanks for highlighting such an important security issue. I will resolve as soon as I can and re-apply for WL ;)

**Pull request closed without merging**